scripts/version.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
```

Comment:
Let's add a license header here.

Reply:
The current version of aep-plugin used by Assurance library does not yet have capability to add license to non-code files. I will update the plugin and apply licenses to all other files in a follow-up pr.